libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -91,6 +148,7 @@ internal class RouteRefreshController(
         } finally {
```

Comment:
Don't you want to do something with the state in the catch block?

Reply:
it's changing to `failed` at line 151 🤔

Reply:
But what if there is an exception? The method will throw and you won't reach line 151, will you?

Reply:
I see 👍 it will be cought above in `refresh()` method, line `79`.
```kotlin
} catch (e: CancellationException) {
    resetState()
    throw e
}
```